osbs/build/build_request.py

```diff
@@ -867,7 +889,13 @@ def render(self, validate=True):
                            # add the path to the plugin's
                            # configuration. This is done elsewhere.
                            None):
-                          self.spec.registry_secrets.value})
+                          self.spec.registry_secrets.value,
+
+                          ('buildstep_plugins', 'orchestrate_build', 'osbs_client_config'):
```

Comment:
Also, can we omit `orchestrate_plugin` from this PR? Currently, this PR shouldn't be merged until the plugin is finalized and merged. This dependency seems unnecessary.